packages/ckb/src/constants/index.ts

```diff
@@ -294,6 +276,33 @@ const MainnetInfo = {
   } as CKBComponents.Script,
 };
 
+export const COMPATIBLE_XUDT_TYPE_SCRIPTS: CKBComponents.Script[] = [
```

Comment:
If `COMPATIBLE_XUDT_TYPE_SCRIPTS` is maintained statically in the rgbpp-sdk, it will come out when a new xUDT compatible token is released, all dApps using rgbpp-sdk will need to update rgbpp-sdk.

Reply:
I tried to fetch compatible xUDT list from GitHub of Vercel server in https://github.com/utxostack/rgbpp-sdk/pull/296.

However, these changes will cause incompatible upgrades. Judging from the current situation of ecological projects, maintaining a static list is an acceptable solution.

Reply:
Is it acceptable to add an async version of the `isUDTTypeSupported` method?

Reply:
@ShookLyngs This is also the issue I want to discuss.

Reply:
Then we need to maintain two functions. Which function do we recommend third-party developers use?

Reply:
Or:
```
isUDTTypeSupported(..., fetchLatest: false): boolean | Promise<boolean>;
isUDTTypeSupported(..., fetchLatest: true): Promise<boolean>;
```

Reply:
Another idea, **for anyone who needs the tip information**, they should fetch the latest version of the scripts before calling any rgbpp-sdk/ckb functions. For example, they can call a function like this:

```ts
await fetchLatestXXXScript(); // call this
await initCkbScripts(); // or this
```

Reply:
> Another idea, **for anyone who needs the tip information**, they should fetch the latest version of the scripts before calling any rgbpp-sdk/ckb functions. For example, they can call a function like this:
> 
> ```ts
> await fetchLatestXXXScripts(network); // call this
> await initCkbScripts(network); // or this
> ```

Is it another big breaking change?

Reply:
> Is it another big breaking change?

I think this is not a breaking change, because not everyone should use it.
Only if the caller strically need the latest version of the scripts, you should use the new function.